generated_code/elastic.py

```diff
@@ -54,6 +55,10 @@ def __init__(self, order, multipleSimulations, matricesDir, memLayout, **kwargs)
 
     memoryLayoutFromFile(memLayout, self.db, clones)
 
+    selectVelocitySpp = np.zeros((self.numberOfQuantities(), 3))
```

Comment:
Same as:
https://github.com/SeisSol/SeisSol/blob/master/generated_code/SurfaceDisplacement.py#L56 

?

Reply:
You could add this in the constructor of LinearADERDG (or whatever the name is) - so in the superclass of elastic, etc. And then remove the call in SurfaceDisplacement and use the one from the super class

Reply:
done